frontend/app/transforms/django-url.js

```diff
@@ -0,0 +1,14 @@
+import Transform from '@ember-data/serializer/transform';
+
+export default class DjangoUrlTransform extends Transform {
+  deserialize(serialized) {
+    return serialized;
+  }
+
+  serialize(deserialized) {
+    if (deserialized && !deserialized.startsWith('http')) {
+      return `https://${deserialized}`;
+    }
+    return deserialized;
+  }
+}
```

Comment:
Drop in favor of proper validation of the form

Reply:
done in https://github.com/adfinis/Outdated/pull/14/commits/7572607b01a33f53638e097b5113ac15566e417e